nnvm/tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -108,32 +108,32 @@ def _test_pooling(input_shape, **kwargs):
 
 def test_forward_pooling():
     """ Pooling """
-
-    _test_pooling(input_shape=[2, 9, 10, 2],
-                 window_shape=[1, 1],
-                 padding='SAME',
-                 pooling_type='MAX',
-                 dilation_rate=[1, 1],
-                 strides=[1, 1])
-
-    _test_pooling(input_shape=[2, 9, 10, 2],
-                 window_shape=[1, 1],
-                 padding='SAME',
-                 pooling_type='AVG',
-                 dilation_rate=[1, 1],
-                 strides=[1, 1])
-    _test_pooling(input_shape=[2, 10, 9, 2],
-                 window_shape=[1, 1],
-                 padding='SAME',
-                 pooling_type='MAX',
-                 dilation_rate=[1, 1],
-                 strides=[1, 1])
-    _test_pooling(input_shape=[2, 10, 9, 2],
-                 window_shape=[1, 1],
-                 padding='SAME',
-                 pooling_type='AVG',
-                 dilation_rate=[1, 1],
-                 strides=[1, 1])
+    for padding in ["SAME", "VALID"]:
+        for pooling_type in ["MAX", "AVG"]:
+            for input_shape in [[2, 9, 10, 2], [2, 10, 9, 2]]:
+                for window_shape in [[1, 1], [2, 1], [2, 3]]:
+                    if padding != "SAME":

```

Comment:
the testing script is not clean. I think it's ok to test a few important cases using the old style.